templates/terraform/resource.erb

```diff
@@ -28,7 +28,7 @@ package google
     client_name_camel = client_name.camelize(:lower)
     client_name_pascal = client_name.camelize(:upper)
     client_name_lower = client_name.downcase
-    has_project = object.base_url.include?('{{project}}')
+    has_project = object.base_url.include?('{{project}}') || object.create_url && object.create_url.include?('{{project}}')
```

Comment:
For clarity in the order these conditions get resolved, can you add parentheses for grouping?